src/entity/user/UserCandidatePreference.ts

```diff
@@ -126,4 +126,7 @@ export class UserCandidatePreference {
     foreignKeyConstraintName: 'FK_user_candidate_preference_user_id',
   })
   user: Promise<User>;
+
+  @Column({ type: 'text', default: null })
```

Comment:
Ah i thought we would just store this in the parsedJson as a string?
Eitherway not too worried about it, guess this is easier cleanup

Reply:
cleaner to keep it separate :)